modules/ocf_mirrors/files/healthcheck

```diff
@@ -19,9 +20,33 @@ import requests
 Mirror = namedtuple('Mirror', ['url', 'updated_at'])
 
 
-def get_updated(mirror_url):
-    """Find the time the host was last updated.
+def update_func(healthcheck):
+    if healthcheck == 'debian':
+        return get_updated_debian
+    elif healthcheck == 'ts':
+        return get_updated_ts
```

Comment:
can you use a better name for ts? even spelling out "unix_timestamp" would be preferable if that's what it means